locksmith/src/websub/helpers/renewFiatKey.ts

```diff
@@ -0,0 +1,163 @@
+import { getStripeConnectForLock } from '../../operations/stripeOperations'
+import Dispatcher from '../../fulfillment/dispatcher'
+import { logger } from '../../logger'
+import { KeyRenewal, KeySubscription } from '../../models'
+import Stripe from 'stripe'
+import config from '../../../config/config'
+import { Op } from 'sequelize'
+import { Web3Service } from '@unlock-protocol/unlock-js'
+import networks from '@unlock-protocol/networks'
+
+interface RenewKeyReturned {
+  keyId?: number
+  lockAddress: string
+  network: number
+  tx?: string
+  error?: string
+}
+
+interface Options {
+  lockAddress: string
+  userAddress: string
+  keyId: number
+  network: number
+}
+
+export async function renewFiatKey({
+  lockAddress,
+  userAddress,
+  keyId,
+  network,
+}: Options): Promise<RenewKeyReturned> {
+  try {
+    const renewalInfo = {
+      keyId,
+      network,
+      lockAddress,
+    }
+
+    const stripe = new Stripe(config.stripeSecret!, {
```

Comment:
Do you think we should have a global helper for this?

Reply:
Please don't make as resolved without at least leaving a response :D

Reply:
github blackholed my response :(

no, I think it is fine for now.